src/js/media-time-range.js

```diff
@@ -408,37 +415,35 @@ class MediaTimeRange extends MediaChromeRange {
   }
 }
 
-function getBoxPosition(el, box, percent) {
-  const rect = el.getBoundingClientRect();
-  const rangeRect = el.range.getBoundingClientRect();
+function getBoxPosition(el, box, ratio) {
+  let position = `${ratio * 100}%`;
+
+  // Get the element that enforces the bounds for the time range boxes.
+  const bounds =
+    (el.getAttribute('media-bounds')
+      ? closestComposedNode(el, `#${el.getAttribute('media-bounds')}`)
```

Comment:
should we rename this attribute `media-bounds`? 
I remember @heff saying `media-` prefixed attrs are propagated. this one is not.

open to suggestions.
(I don't think anyone has discovered this functionality yet)

Reply:
Where is media-bounds coming from?

Reply:
it's used when the media-time-range has a left offset for example
https://media-theme-neo.vercel.app/

the attribute should be the `id` of the container that act as the bounding box for the time range boxes (preview thumb)

Reply:
ah, I see. It's a bit like the thing we talked about for the fullscreen button/media-controller thing.